bundle/Resources/config/routing.yml

```diff
@@ -0,0 +1,4 @@
+ez_content_create_no_draft:
+    path: /content/create/nodraft/{contentTypeId}/{language}/{parentLocationId}
```

Comment:
> And "create" might kind of imply "nodraft" right?

Hmm does it ? In the PAPI at least it implies creating a draft, if I am not mistaken. 
Semantically speaking, it would be createAndPublish, as it does both in one operation.